src/main/java/com/health/BMI_check/controller/BodyDataRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.health.BMI_check.controller;
```

Comment:
いまさらですがパッケージ名は小文字のみが慣習ですね。
```suggestion
package com.health.bmi_check.controller;
```

Reply:
[5fd43df](https://github.com/teddypr/bmi-checker/pull/12/commits/5fd43df5960858711fd09cabad9279f28af3c873)
パッケージ名を小文字に修正しました！